public/scripts/app.js

```diff
@@ -0,0 +1,151 @@
+document.addEventListener("DOMContentLoaded", function() {
+  var calc = new Calculator();
+  var calcButtons = document.getElementsByClassName('calc-display-num')[0];
+  calcButtons.addEventListener('click', function(event) {
+    calc.buttonPressed(event);
```

Comment:
Passing just the `innerText` here, instead of the `event` itself, would make it easier to reuse the `buttonPressed` function later when you add a listener for keyDown events.